src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -192,9 +192,8 @@ namespace OpenLoco::Ui::WindowManager
             [](registers& regs) FORCE_ALIGN_ARG_POINTER -> uint8_t {
                 registers backup = regs;
                 const char* buffer = (const char*)regs.esi;
-                auto& drawingCtx = Gfx::getDrawingEngine().getDrawingContext();
-                auto tr = Gfx::TextRenderer(drawingCtx);
-                uint16_t width = tr.getStringWidth(buffer);
+                // NOTE: Font was previously from the global, assume its always medium_bold for the time being.
+                uint16_t width = Gfx::TextRenderer::getStringWidth(Gfx::Font::medium_bold, buffer);
```

Comment:
`0x00495685` is the vanilla `getStringWidth` function. I don't think anything is still relying on this hook, tbh. Fine to leave it for now, though.

Reply:
I just wanted to eliminate all the useless references to getDrawingContext() so it ended up in the list of changes.